### PR TITLE
changing user's profile name

### PR DESCRIPTION
**Pull request by maximt**
Allows you to change your name in profile (not "username").

https://github.com/spaceshelter/orbitar/assets/230060/9c8b5895-f286-47e7-a044-41025395fc4b

### CODE REVIEW

File: backend/src/api/UserController.ts [hunk 1]
```diff
@@ -65,6 +67,10 @@ export default class UserController {
             bio: Joi.string().required().max(1024 * 4)
         });
 
+        const nameSchema = Joi.object<UserSaveNameRequest>({
+            name: Joi.string().required()
```
Comment (Aivean) [inline backend/src/api/UserController.ts]: Probably a good idea to add a length check here (and also in https://github.com/spaceshelter/orbitar/blob/df2e5f892c9bf810086a117908677b2b96a66d70/backend/src/api/InviteController.ts#L50).

`name` is currently a `varchar(100)`

File: frontend/src/Components/UserProfileName.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import {ReactComponent as SendIcon} from '../Assets/send.svg';
+import {useAPI} from '../AppState/AppState';
+import {toast} from 'react-toastify';
+
+type UserProfileNameProps = {
+  name: string;
+  mine: boolean;
+};
+
+export default function UserProfileBio(props: UserProfileNameProps) {
```
Comment (Aivean) [inline frontend/src/Components/UserProfileName.tsx]: Did you mean `UserProfileName`?

**Review by Aivean — DISMISSED** [review]
Minor nits, otherwise looks good.

File: backend/src/api/UserController.ts (hunk 1 above)
Comment (maximt) [reply backend/src/api/UserController.ts]: done

**Review by Aivean — APPROVED** [review]

**Pull request merged**